### PR TITLE
intialize project

### PR DESCRIPTION
**Pull request by jleber**
# :construction: Project Creation

This first PR contains the base files for the gem project.

# :repeat: Steps to test

In your terminal at the project root, run:

```bash
    bundle install
    gem build pipefy_message.gemspec
    gem install pipefy_message-0.1.0.gem
    
    require 'pipefy_message'
    message = PipefyMessage::Test.new
    message.hello
```

# 🗂 Related cards

[[async] Create and setup a project for an internal gem](https://app.pipefy.com/open-cards/490511375)

### CODE REVIEW

**Comment by viniciuscbm** [comment]
What do you think about getting the moment and adding a pull request template? It's will help to understand this pull request and the next.

If you like this idea this link shows how to do it: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository

```
# ✨ New Feature

Describe what is the new feature/piece of code that you've introduced to the repo.

Simply describe the new feature. Leave the full description to the card.

# :repeat: Steps to reproduce

Is there any particularity that's worth to mention in order to be able to test this feature in a local environment?

If so, please share with the reviewers in this section.

# 🖼 Screenshots

| Before 🐛 | After ✨ |
| --------- | -------- |
|           |          |

# 🗂 Related cards

[Card Title](https://app.pipefy.com/open-cards/cardId)
```

File: .gitignore [hunk 1]
```diff
@@ -54,3 +54,5 @@ build-iPhoneSimulator/
 
 # Used by RuboCop. Remote config files pulled in from inherit_from directive.
 # .rubocop-https?--*
+
+.idea
```
Comment (viniciuscbm) [inline .gitignore:58]: Ruby and Rubocop have a pattern to use the last line of every file how is a blank line, WDYT team, we should follow it or not? :nail_care:

File: .rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+AllCops:
+  TargetRubyVersion: 2.6
```
Comment (viniciuscbm) [inline .rubocop.yml:2]: Maybe is better to use ruby in version 7.1, WDYT?

File: README.md [hunk 3]
```diff
@@ -1 +1,43 @@
-# pipefy_message
\ No newline at end of file
+# PipefyMessage
```
Comment (viniciuscbm) [inline README.md:1]: How this project doesn't have docker, I think is a good idea to specify the version of Ruby that is necessary to run locally.

**Comment by jleber** [comment]
> What do you think about getting the moment and adding a pull request template? It's will help to understand this pull request and the next.
> 
> If you like this idea this link shows how to do it: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository
> 
> ```
> # ✨ New Feature
> 
> Describe what is the new feature/piece of code that you've introduced to the repo.
> 
> Simply describe the new feature. Leave the full description to the card.
> 
> # :repeat: Steps to reproduce
> 
> Is there any particularity that's worth to mention in order to be able to test this feature in a local environment?
> 
> If so, please share with the reviewers in this section.
> 
> # 🖼 Screenshots
> 
> | Before 🐛 | After ✨ |
> | --------- | -------- |
> |           |          |
> 
> # 🗂 Related cards
> 
> [Card Title](https://app.pipefy.com/open-cards/cardId)
> ```

@viniciuscbm I create this [PR](https://github.com/pipefy/pipefy_message/pull/3) with the suggested template, could you take a look ?

File: .gitignore (hunk 1 above)
Comment (jleber) [reply .gitignore:58]: it's is applied to .gitignore too ?

Comment (viniciuscbm) [reply .gitignore:58]: I don't know, but every file without a last blank line will appear like this:

image

Some projects aren't acetable, my question is about if we will don't care about this or if will put an end line. The VS Code has plugins to do it.

File: .rubocop.yml (hunk 2 above)
Comment (jleber) [reply .rubocop.yml:2]: It's possible to have any conflicts with pipefy-core using this version?

If not it's ok for me 👍🏾

Comment (viniciuscbm) [reply .rubocop.yml:2]: lol, I put the version o ES ahuahuauha

Sorry, I talk about the Ruby 3.1.0

File: README.md (hunk 3 above)
Comment (jleber) [reply README.md:1]: sure, we just have to choose the version that we'll use and then I update the readme

File: .rubocop.yml (hunk 2 above)
Comment (viniciuscbm) [reply .rubocop.yml:2]: The pipefy core use ruby 2.6.6

Comment (viniciuscbm) [reply .rubocop.yml:2]: You are correct, the max version we could use here is 2.6.6

File: README.md (hunk 3 above)
Comment (viniciuscbm) [reply README.md:1]: I think version 2.6.6

File: .gitignore (hunk 1 above)
Comment (jleber) [reply .gitignore:58]: I tested here and Rubocop only analyzed Ruby source files, so my answer is for this kind of file I agree to use this pattern, but for the non-ruby files, I think it's not necessary.

WDYT ?

File: README.md (hunk 3 above)
Comment (jleber) [reply README.md:1]: ok, I'll create a section for that

File: .gitignore (hunk 1 above)
Comment (viniciuscbm) [reply .gitignore:58]: It's ok to me 😁

File: README.md (hunk 3 above)
Comment (jleber) [reply README.md:1]: done

**Review by viniciuscbm — APPROVED** [review]
Nice, ship it! 🚀

**Pull request merged**